### PR TITLE
docs(FAQ): change version of yarn to v4

### PR DESCRIPTION
**Pull request by langz**
We could perhaps change/update some other information on this page as well?
I guess somethings are outdated.

https://eufemia.dnb.no/contribute/faq/

### CODE REVIEW

**Comment by tujoworker** [comment]
We could also remove the version notation actually.

File: packages/dnb-design-system-portal/src/docs/contribute/faq.mdx [hunk 1]
```diff
@@ -27,7 +27,7 @@ Due to this bug: https://github.com/babel/babel/issues/11394 we add `.png,.snap`
 
 ### Yarn PnP
 
-Currently, Eufemia uses yarn v3 with `node_modules`.
+Currently, Eufemia uses yarn v4 with `node_modules`.
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/contribute/faq.mdx:30]:
```suggestion
Currently, Eufemia uses yarn with `node_modules`.
```

**Comment by langz** [comment]
> We could also remove the version notation actually.

Sure, like this?
https://github.com/dnbexperience/eufemia/pull/2944/commits/f341d3d3295e106154cc077182b9cca5fd68eac7

**Comment by tujoworker** [comment]
yes.

**Pull request merged**